### DISCUSSION TITLE
Backfill partitioned asset with a run configuration

### DISCUSSION THREAD

**Discussion by stevenmurphy12** [dagster-io/dagster#19352] [Q&A] [answered]
When I materialise a partitioned asset through the UI, I have the option of adding run configuration in the launchpad.

When I select multiple partitions, I only have the option to backfill, I cannot add custom run configuration.

Is it possible to run a backfill with run configuration?

**Comment by garethbrickman** [comment, marked as answer]
I think this may duplicate/relate to this issue: https://github.com/dagster-io/dagster/issues/10079
